recipes-extended/xen/files/libxl-openxt-helpers.patch

```diff
@@ -195,22 +197,3 @@ PATCHES
  #define ROUNDUP(_val, _order)                                           \
      (((unsigned long)(_val)+(1UL<<(_order))-1) & ~((1UL<<(_order))-1))
  
---- a/tools/libxl/libxl_create.c
-+++ b/tools/libxl/libxl_create.c
-@@ -1570,12 +1570,10 @@ static void domcreate_devmodel_started(l
-     if (dcs->sdss.dm.guest_domid) {
-         if (d_config->b_info.device_model_version
-             == LIBXL_DEVICE_MODEL_VERSION_QEMU_XEN) {
--            if (!libxl_defbool_val(d_config->b_info.device_model_stubdomain)) {
--                libxl__qmp_initializations(gc, domid, d_config);
--            } else {
--                int stubdom_domid = dcs->dmss.pvqemu.guest_domid;
--                libxl__qmp_initializations(gc, stubdom_domid, d_config);
--            }
-+            /* OpenXT: for stubdom-enabled guests, qmp_helper creates the same
-+             * unix socket than QEMU would for non-stubdom guests.
-+             * Therefore there's no need to dissociate the two cases */
-+            libxl__qmp_initializations(gc, domid, d_config);
-         }
-     }
- 
```

Comment:
There are a lot more references to the qmp helper in this patch. Are we leaving that logic intact intentionally?

Reply:
I think you are right that qmp-helper reference can be removed since we are using vchan-socket-proxy.  If qmp-helper wants to be resurrected, it can be done by specifying via `configure --stubdom-qmp-proxy` and making it compatible argument wise.

Reply:
It is okay to remove qmp-helper from libxl-openxt-helpers.patch in its own patch closer to the end of the queue?  I'd rather not have to deal with rebasing and all the patch offset conflicts if I don't have to.  A repo of patches doesn't rebase easily.

Reply:
I removed qmp-helper earlier in the series and then fixed up from there.  Some of the intermediate points might be slightly off for their patch offsets.